src/sync.js

```diff
@@ -290,11 +284,13 @@ function retrieveNote(client) {
  *   last_modified: the timestamp of the sync, or null
  * }
  */
-function loadFromKinto(client, credentials) {
+function loadFromKinto(client, credentials) { // eslint-disable-line no-unused-vars
+  console.log('loadFromKinto', client, credentials);
```

Comment:
Nit: not sure if we want to remove these debugging `console.log()` messages (here and L293 below).

Reply:
😆, definitely should be removed. Was debugging late and and forgot those, thanks 👍